src/app/cardsView/services/project/project-mock.service.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Created by wiekonek on 21.11.16.
+ */
+import {Injectable} from "@angular/core";
+import {Http} from "@angular/http";
+import {ProjectService} from "./project.service";
+import {Card} from "../../../card/card";
+import {Observable} from "rxjs";
+
+@Injectable()
+export class ProjectsMockService extends ProjectService {
```

Comment:
ten serwis jest odpowiedzialny za pobieranie mockowych danych, tak? czyli w zależności od tego, czy ktoś uruchomi produkcję, czy może development, to inny serrwis będzie wstrzykiwany? czy może w innym celu pobieramy mockowe dane?